fix wording + small improvements to Template Apps page (#1831)
mongodb/docs-realm: source/manage-apps/create/template-apps.txt

.. tabs::
   
   .. tab:: {+ui+}
      :tabid: ui
      
      Follow the procedure described in the 

>       :ref:`Create a Realm App with Realm UI <create-a-realm-app>` guide, and click

Reviewer: ```suggestion
      :ref:`Create a Realm App with Realm UI <create-a-realm-app>` guide, and select
```

more general to accommodate screen readers/mobile, etc

      :tabid: api

      Follow the procedure described in the 

>       :ref:`Create a Realm App with Realm API <create-app-api>` guide, and click

Reviewer: ```suggestion
      :ref:`Create a Realm App with Realm API <create-app-api>` guide, and select
```

same as above

1. Learn more about what your chosen template app includes.

2. Follow on-screen guides for customizing your app.

3. Use the {+cli-bin+} to download the code for your app so you can build the frontend 
   (if one is included) and interact with the {+app+} through the cli. 

>    If you already have {+cli-bin+} setup, you can use the {+cli-bin+}

Reviewer: ```suggestion
   If you already have {+cli-bin+} set up, you can use the {+cli-bin+}
```

just did quick goog on this. [only 1 word if noun](https://grammarist.com/spelling/set-up-vs-setup/#:~:text=Setup%20is%20one%20word%20when,set%20up%20the%20computer%E2%80%9D)
